Frontend/src/pages/register/components/RegistrationPanel.jsx

```diff
@@ -152,7 +162,11 @@ export default function RegistrationPanel() {
             value={guests}
             onChange={(e, data) => setGuests(data.value)}
             selection
-            options={[...new Array(10)].map((_, index) => {
+            options={[
+              ...new Array(
+                (competitionInfo.guests_per_registration_limit ?? 99) + 1 // Arrays start at 0
+              ),
+            ].map((_, index) => {
```

Comment:
I think you can just `options = new Array(...).map(...)`.

Reply:
you would think so, but otherwise you get a sparse array with 99 empty slots, which you can't map on